libnopegl/src/node_resourceprops.h

```diff
@@ -23,7 +23,7 @@
 #define NODE_RESOURCEPROPS_H
 
 struct resourceprops_opts {
-    int precision;
+    enum ngpu_precision precision;
```

Comment:
`ngpu_precision` is not declared in this context, can you add the appropriate header?